.gitlab-ci.yml

```
@@ -16,7 +16,8 @@ linux:
   script:
     - curl -sSL https://get.docker.com/ | sh
     - docker run --rm --privileged docker.io/tonistiigi/binfmt:latest --install all
-    - python -m pip install -e ".[dev]" pytest-custom-exit-code
+    - python -m pip install dependency-groups
+    - python3 -m dependency_groups test | xargs python -m pip install -e. pytest-custom-exit-code
```

```suggestion
    - python -m dependency_groups test | xargs python -m pip install -e. pytest-custom-exit-code
```
It's working with either `python` or `python3` but better stay consistent here.